lib/exports/colors-new/theme.less

```diff
@@ -0,0 +1,86 @@
+.theme-variables() {
```

Comment:
For consistency should we add the `-new` suffix to these variables as well?
(of course we will need to drop it in the next story but it might be a good reminder that we will actually have to add the `legacy` suffix to the old theme variables).

What is the reason we haven't switched to new color variables here? 
Do we need clarification from design?

Reply:
I don't think these theme variables should get `new` or `legacy` suffixes, since they never define color values; they only reference them. What I decided to do in https://github.com/StackExchange/Stacks/pull/1415/commits/afa19ede76398b3b75ab99a245082c2d690bac68 is just remove duplicates of these variables so they're only generated from this file.

I see switching the values of these variables to the new color variables here as work to when we get to "[Update Stacks Components Styles to use new Color Palette](https://stackoverflow.atlassian.net/browse/STACKS-349)".

Reply:
Understood, it makes sense. 👍